src/core/mac/sub_mac.cpp

```diff
@@ -966,12 +984,14 @@ void SubMac::HandleCslTimer(Timer &aTimer)
 
 void SubMac::HandleCslTimer(void)
 {
+    uint32_t duration;
+
```

Comment:
Maybe we can put a diagram here for better understanding

```suggestion
    /*
     * CSL sample timing illustration
     * Maybe we can put a diagram here for better understanding

```suggestion
    /*
     * CSL sample timing illustration
     *      |<-------------------------------------Wake----------------------------------------->|<------------Sleep------------->|
     *      |                                                                                    |                                |
     *      |<--Ahead-->|<--UnCert-->|<--ClockDrift-->|<--ClockDrift-->|<--UnCert-->|<--MinWin-->|                                |
     *      |           |            |                |                |            |            |                                |
     * -----|-----------|------------|----------------|----------------|------------|------------|---------------//---------------|------
     * SampleStart                                 CslPhase                                  SampleEnd                        SampleStart
     */
    uint32_t duration;

```
     */
    uint32_t duration;

```